src/main/java/org/mitre/synthea/engine/Transition.java

```diff
@@ -86,7 +98,7 @@ public static final class DistributedTransitionOption extends TransitionOption {
    * have an effective distribution of 0.25, and the last transition will have an
    * effective distribution of 0.0).
    */
-  public static final class DistributedTransition extends Transition {
+  public static class DistributedTransition extends Transition {
```

Comment:
It's not needed for this PR. I originally removed it because an initial approach worked by subclassing `DistributedTransition`. That said, there is no real reason to mark this class as `final`. I asked @dehall as I believe he was the person who initially marked it as `final` and he could not remember a good reason for doing so.